Update README.md (#1719)
DataDog/integrations-extras: twingate/README.md

4. Restart the Datadog Agent by running `service datadog-agent restart`.
5. Confirm that the Twingate Analytic log appears in the [Log Explorer][10].

### Setup Datadog Log Pipeline
1. Navigating to **Logs** > **Pipeline**
2. Search for Twingate

> 3. Clone the pipeline

Reviewer: These aren't the steps for setting up a pipeline, but rather for creating a new pipeline to customize. Would you be open to expanding upon the sorts of customizations that users might want to make in this case?

Author: Hi @buraizu, thank you for your feedback.

the pipeline is already part of the Datadog pipeline library and no additional customisation needed.

Reviewer: Thanks for confirming that. As such, the pipeline should be automatically enabled when logs arrive with the designated `source` attribute, as mentioned in the documentation here:
https://docs.datadoghq.com/logs/log_configuration/pipelines/?tab=source#integration-pipelines
The steps provided would create a clone of the existing pipeline in order to make modifications. Since you're not suggesting any modifications to the existing pipeline, this section can be removed.

> ### Setup Datadog Log Pipeline
> 1. Navigating to **Logs** > **Pipeline**
> 2. Search for Twingate
> 3. Clone the pipeline

Reviewer: ```suggestion
### Setup Datadog Log Pipeline
```

> ### Datadog Dashboard
> 1. Navigating to **Dashboards** > **Dashboard List**
> 2. Select Twingate Analytics Dashboard

Reviewer: ```suggestion
### Twingate Analytics Dashboard
1. Go to the Datadog [Dashboard List][12].
2. Search for the Twingate Analytics dashboard.
```

[8]: https://raw.githubusercontent.com/Twingate-Labs/datadog-integrations-extras/master/twingate/images/dashboard.png
[9]: https://docs.datadoghq.com/logs/explorer/facets/#measures
[10]: https://app.datadoghq.com/logs

> [11]: https://docs.datadoghq.com/getting_started/agent/

Reviewer: ```suggestion
[11]: https://docs.datadoghq.com/getting_started/agent/
[12]: https://app.datadoghq.com/dashboard/lists
```
